api/authentication/authentication.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const router  = express.Router();
+const Parse = require('parse/node')
+
+// Register route creates a user in the database, displays an error message
+// if the user cannot be created.
+router.post('/register', async (req, res) => {
+    let user = new Parse.User(req.body)
+  
+    try {
+        await user.signUp()
+        res.status(201)    
+        res.send({"user" : user})
+    } catch (error) {
+        res.status(400)
+        res.send({"error" : "Failed to create user: " + error })
+    }
+  })
+  
+  
+  // Logs in user by accessing database, displays error message if log in fails
+  router.post('/login', async (req, res) => {
+    try {
+      const user = await Parse.User.logIn(req.body.username, req.body.password)
+      res.send({"user" : user})
+    } catch (error) {
+      res.status(400)
+      res.send({"error" : "Login failed: " + error })
+    }
+  })
+  
+  
+  // gets messages from DB, this route will be replaced with info from RealtyMole
+  router.get('/messages', async (req, res) => {
+    try {
+      const query = new Parse.Query("Messages")
+  
+      query.descending("createdAt")
+      query.include("user")
+  
+      messages = await query.find()
```

Comment:
nit: 

`const messages = await query.find()`

FYI: "nit" is common to preface messages in code review. -meaning the reviewer acknowledges this is a "nitpicky" comment